include/mvd/bits/sonata_misc.hpp

```diff
@@ -172,6 +172,12 @@ inline std::vector<std::string> SonataFile::getSynapseClass(const Range & range)
     return pop_->getAttribute<std::string>(did_synapse_class, select(range, size_));
 }
 
+inline std::vector<TSV::TSVInfo> SonataFile::getTSVInfo(const Range & range ) const{
```

Comment:
```suggestion
inline std::vector<TSV::TSVInfo> SonataFile::getTSVInfo(const Range& range) const {
```

Reply:
I kept the style of the previous functions declarations but I see that the next declarations follow a different style as well with the `&` together with `range`. Should in every function this be changed to what you pointed out?